ipaserver/install/kra.py

```diff
@@ -69,9 +68,19 @@ def install_check(api, replica_config, options):
 
 
 def install(api, replica_config, options):
-    if replica_config is None:
-        if not options.setup_kra:
-            return
+
+    if replica_config is not None and not replica_config.setup_kra:
+        return
+
+    install_kra_replica = False
+    kra = krainstance.KRAInstance(api.env.realm)
+
+    if kra.is_installed() and replica_config is not None:
+        install_kra_replica = False
+    elif not kra.is_installed():
+        install_kra_replica = True
+
```

Comment:
So actually thinking about this check more I am not sure why we need to check if replica_config is None.
Either a kra is installed, in which case we need to make a clone (replica) or the kra is not installed in which case we need to create one from scratch.
however this check is wrong, because "is_installed" checks if a KRA is installed locally, not in the domain.

So I think this check is completely useless. Remove it.